extractor/src/main/java/org/schabi/newpipe/extractor/stream/Stream.java

```diff
@@ -1,13 +1,14 @@
 package org.schabi.newpipe.extractor.stream;
 
-import org.schabi.newpipe.extractor.MediaFormat;
-
 import java.io.Serializable;
 import java.util.List;
 
+import org.schabi.newpipe.extractor.MediaFormat;
+
 public abstract class Stream implements Serializable {
```

Comment:
Could you plese add a jdoc documentation to this file, and explain the use of torrenturl :)

Reply:
done.
added example of the optional parameter `torrentUrl` (https://webtorrent.io/torrents/big-buck-bunny.torrent), supplied if the stream can be played via bittorrent.